frontend/src/components/Psychologist/Profile/InactiveProfile.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { TextInput, RadioGroup, Radio, Button, Alert, ButtonGroup } from '@dataesr/react-dsfr';
+import agent from 'services/agent';
+import Page from 'components/Page/Page';
+
+const InactiveProfile = () => {
+  const [reason, setReason] = useState();
+  const [displayReason, setDisplayReason] = useState(false);
+  const [otherReason, setOtherReason] = useState('');
+  const [done, setDone] = useState(false);
+  const { token } = useParams();
+
+  const getReason = () => (reason === 'other' ? `Autre: ${otherReason}` : reason);
+
+  const selectReason = value => {
+    if (value !== 'other') {
+      setDisplayReason(false);
+      setReason(value);
+    }
+  };
+  const canValidate = reason
+  && (reason !== 'other' || otherReason.trim() !== '');
+
+  return (
+    <Page
+      background="blue"
+    >
+      {done && (
+        <Alert
+          className="fr-mb-2w"
+          type="success"
+          {...done}
+        />
+      )}
+      <RadioGroup
+        legend="Vous nous avez indiqué vouloir quitter le dispositif SPE. Afin d’améliorer le dispositif Santé Psy Etudiant, pouvez-vous nous indiquer les raisons de ce choix :"
```

Comment:
Update message to avoid redundancy. Message proposed by Paul in Mattermost.

Reply:
I didn't find the message by Paul but i removed the redundancy